optimize/util/optimize-commons/src/main/java/io/camunda/optimize/dto/optimize/query/event/DeletableEventDto.java

```diff
@@ -1,39 +0,0 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Camunda License 1.0. You may not use this file
- * except in compliance with the Camunda License 1.0.
- */
-package io.camunda.optimize.dto.optimize.query.event;
-
-import io.camunda.optimize.dto.optimize.query.event.process.EventDto;
-import java.time.Instant;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
-import lombok.experimental.FieldNameConstants;
-
-@Data
-@AllArgsConstructor
-@NoArgsConstructor
-@FieldNameConstants
-public class DeletableEventDto {
```

Comment:
Generally speaking, can we mark the REST docs internally as C7 for the ones we are removing from C8?

Reply:
Yes, but you will probably have to remind me every time

Reply:
I think we should actually have two sets of docs tbh, this might be something we do before handover